cluster-provision/gocli/opts/cnao/cnao.go

```diff
@@ -46,10 +48,16 @@ func (o *cnaoOpt) Exec() error {
 					continue
 				}
 
-				if path == "manifests/network-addons-config-example.cr.yaml" && o.multusEnabled {
-					re := regexp.MustCompile("(?m)[\r\n]+^.*multus.*$")
-					res := re.ReplaceAllString(string(yamlDoc), "")
-					yamlDoc = []byte(res)
+				if path == "manifests/network-addons-config-example.cr.yaml" {
+					if o.skipCR {
```

Comment:
one question, there is the code that deploys multus V3, it should also respect this flag isnt it ?
unless done already

Reply:
you mean if this flag is passed, multus deployment should also be skipped even if `multusEnabled` itself is true ?

Reply:
i see. well in the past no, with_multus_v3 was stronger. even if you said to skip the CNAO CR but said you wanted multus v3 it would get deployed (but with its own crds)
if you would like to change this behavior we can include this change in the PR

Reply:
yep you right about the past
well lets keep as it was, it gives bit more flexibility, and doesnt change behavior
unless you think otherwise?  (we can always change in the future according needs)
thanks